src/Components/List/List.jsx

```diff
@@ -37,21 +37,55 @@ export default function List({ items, token }) {
     history.push('/add-item');
   };
 
+  const [searchItem, setSearchItem] = React.useState('');
+  const handleChange = (e) => {
+    setSearchItem(e.target.value);
+  };
+
+  const resetSearch = () => {
+    setSearchItem('');
+  };
+
+  const results = !searchItem
+    ? items
+    : items.filter((item) =>
+        item.name.toLowerCase().includes(searchItem.toLocaleLowerCase()),
+      );
+
   return (
     <div className="List">
       {items.length === 0 ? (
         <section className="listContainer emptyList">
           <h3>
             Your shopping list is empty. Add a new item to start your list.
           </h3>
-          <button onClick={redirectPath}>Add New Item</button>
+          <button className="emptyButton" onClick={redirectPath}>
+            Add New Item
+          </button>
         </section>
       ) : (
-        <section className="listContainer">
-          <h3>Item List:</h3>
+        <section className="listContainer populatedList">
```

Comment:
This isn't blocking, but I wonder if it might be a good idea for us to refactor this component to use more semantic HTML elements like `section` for the search and the `itemsList`, and `main` for `listContainer`.